package.json

```diff
@@ -120,6 +120,7 @@
   },
   "devDependencies": {
     "@next/bundle-analyzer": "9.4.1",
+    "@nordnet/cz-conventional-changelog": "^1.1.0",
```

Comment:
I advise you to `npm config set save-exact true` to use fixed versions by default. Mostly for a security/stability standpoint.

https://stackoverflow.com/questions/30656189/make-npm-install-save-add-a-strict-version-to-package-json

Reply:
make sense.